edx_exams/apps/lti/tests/test_views.py

```diff
@@ -155,9 +387,9 @@ def test_start_proctoring_unauthorized_user(self, mock_get_lti_launch_url):  # p
 
 @patch('edx_exams.apps.lti.views.get_lti_1p3_launch_start_url', return_value='https://www.example.com')
 class LtiEndAssessmentTestCase(ExamsAPITestCase):
-    """
+    '''
```

Comment:
Convention across our repos has docstrings with double quotes so I'd recommend we stick with that.